src/nmark/def.nim

```diff
@@ -177,21 +177,19 @@ const
   reLinkRef = re" {0,3}\[\s*.*\s*]:(\s*\n?\s*)"
   reEntity* = re"&[a-zA-Z0-9#]+;"
 
-const olNum* = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
-const puncChar* = ['!', '"', '#', '$', '%', '&', '\'', '(', ')', '+', ',', '-', '.', '/', ':', ';', '<', '=', '>', '?', '@', '[', '\\', ']', '^', '`', '{', '|', '}', '~']
-const unchangedChar* = ['!', '#', '$', '%', '\'', '(', ')', '+', ',', '-', '.', '/', ':', ';', '=', '?', '@', '[', '\\', ']', '^', '`', '{', '|', '}', '~', '_', '*']
+const olNum* = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9'}
+const puncChar* = {'!', '"', '#', '$', '%', '&', '\'', '(', ')', '+', ',', '-', '.', '/', ':', ';', '<', '=', '>', '?', '@', '[', '\\', ']', '^', '`', '{', '|', '}', '~'}
+const unchangedChar* = {'!', '#', '$', '%', '\'', '(', ')', '+', ',', '-', '.', '/', ':', ';', '=', '?', '@', '[', '\\', ']', '^', '`', '{', '|', '}', '~', '_', '*'}
 
 proc delWhitespace*(line: string): string =
+  result = newStringOfCap(line.len div 3 * 2)
   for c in line:
-    if c != ' ' and c != '\t': result.add(c)
-  return result
+    if c notin {' ', '\t'}: result.add(c)
```

Comment:
This makes the code a bit easier to read, but I think it would be around the same regarding performance